src/index.html

```diff
@@ -3,10 +3,439 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops & computers"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <article class="card catalog__card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h1 class="card__name">
```

Comment:
will be better to use h2 or h3 here
h1 is used for the main title of the page, and it should be only one on the page